### PR TITLE
feat: add pymanifest parser

### PR DESCRIPTION
**Pull request by ObserverOfTime**
This one targets `main`.

### CODE REVIEW

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -1296,6 +1296,15 @@ M.configs = {
     tier = 2,
   },
 
+  pymanifest = {
+    install_info = {
+      url = 'https://github.com/ObserverOfTime/tree-sitter-pymanifest',
+      files = { 'src/parser.c' },
+    },
+    maintainers = { '@ObserverOfTime' },
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:1304]:
```suggestion
    maintainers = { '@ObserverOfTime' },
    tier = 2,
```

**Comment by ObserverOfTime** [comment]
I'll rebase my other PRs afterwards. We can add those to both.

**Comment by ObserverOfTime** [comment]
I don't know, whatever works. :thinking:

**Comment by clason** [comment]
Can target `master` now :)

**Pull request closed without merging**